src/compiler/checker.ts

```diff
@@ -19295,7 +19322,7 @@ namespace ts {
             if (node.expression.kind === SyntaxKind.SuperKeyword) {
                 const superType = checkSuperExpression(node.expression);
                 if (isTypeAny(superType)) {
-                    forEach(node.arguments, checkExpresionNoReturn); // Still visit arguments so they get marked for visibility, etc
+                    forEach(node.arguments, checkExpressionNoReturn); // Still visit arguments so they get marked for visibility, etc

```

Comment:
This seems like a complex way to avoid writing a `for` loop...

Reply:
Yep!